systemvm/debian/opt/cloud/bin/merge.py

```diff
@@ -236,7 +236,7 @@ def processCLItem(self, num, nw_type):
                 if('localgw' in self.qFile.data['cmd_line']):
                     dp['gateway'] = self.qFile.data['cmd_line']['localgw']
                 else:
-                    dp['gateway'] = 'None'
+                    dp['gateway'] = ''
```

Comment:
Why not use None instead of a empty string? Because None is the best way in Python to tell it's empty and does not contain anything.

Reply:
yes, or 'None' (in a string) I thought I commented that but cannot find my remarks.

Reply:
ah madit on the ticket instead of here #2805

Reply:
@wido later on code, there is a string literal concatenation, so if we use None (Type), will raise an "TypeError: cannot concatenate 'str' and 'NoneType' objects".

In python the '' also represent emptyness in if steatements, for this reason in this particular case i`ve used  '' instead None (type) .

Reply:
ok, good point. So either this (less code) or a check on the gateway later to prevent the faulty concat. I can  live with this solution. @wido?